### DISCUSSION TITLE
pcap.h: No such file or directory error

### DISCUSSION THREAD

**Discussion by zealsham** [projectdiscovery/naabu#139] [Q&A] [answered]
**Describe the bug**
i keep getting this error when i try to install naabu on my vps work/pkg/mod/github.com/google/gopacket@v1.1.19/pcap/pcap_unix.go:34:10: fatal error: pcap.h: No such file or directory

**Comment by ehsandeep** [comment, marked as answer]
@zealsham, you need to install `apt install -y libpcap-dev` before installing naabu.

**Reply by zealsham** [reply]
thanks , worked like charm

**Comment by khalili-mohammad** [comment]
![naabu](https://github.com/projectdiscovery/naabu/assets/108519212/f4e3948a-6874-479e-acf3-4b400a1283d8)
I cannot install it! Is this a bug, or do I need to do something else?
